src/Console/Command/MergeConflictsCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class MergeConflictsCommand extends AbstractNeedApplyCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('merge-conflicts')
+            ->setDescription('Comments non-mergeable pull requests, asking the author to solve conflicts.')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        foreach ($this->configs['projects'] as $name => $projectConfig) {
+            try {
+                $package = $this->packagistClient->get(static::PACKAGIST_GROUP.'/'.$name);
+                $this->io->title($package->getName());
+                $this->checkPullRequests($package);
+            } catch (ExceptionInterface $e) {
+                $this->io->error('Failed with message: '.$e->getMessage());
```

Comment:
Not sure dropping the exception stack trace is a good thing… isn't their a dedicated method to log exceptions in the `$this->io` object?